guide/C/ch_loans.xml

```diff
@@ -639,7 +638,7 @@ The simple case for when  PMT == 0 gives the solution:
       <title>Accounts for the loan</title>
```

Comment:
No. I don't.

The task element can contains title. But it can't contain para, screen, which they are used now.
Its children are blockinfo, example, indexterm, procedure, taskprerequisites, taskrelated, tasksummary, title, titleabbrev.
None of them  match this context.

Reply:
I think a task element doesn't match this context, but example may match it. I will reconsider markups to make translation easy.

Reply:
226e9a9

Reply:
That would mean: convert the last sections into `<step>`s of the `<procedure>` and rework the beginning into `<tasksummary`>, (optional) `<teaskrelated>` and `<taskprerequisites>`.

As I see it seems to be not trivial. it is better done in a separate PR

Reply:
@fellen 
I see what you say.
There are many places that should be changed to task, procedure, example, variablelist, etc. in order for them to match docbook definition.
But I don't have enough resoures to add major changes to the guide.
It will be after translating the concepts guide and help manual. 

https://github.com/Gnucash/gnucash-docs/blob/d989dc0d8c6eedda0eacc997ebd2dbab180f9813/guide/C/ch_invest.xml#L711

https://github.com/Gnucash/gnucash-docs/blob/d989dc0d8c6eedda0eacc997ebd2dbab180f9813/guide/C/ch_invest.xml#L395

https://github.com/Gnucash/gnucash-docs/blob/d989dc0d8c6eedda0eacc997ebd2dbab180f9813/guide/C/ch_txns.xml#L606